cmd/nodecmd/create.go

```diff
@@ -650,31 +678,47 @@ func setCloudInstanceType(cloudService string) (string, error) {
 	return nodeType, nil
 }
 
-func printResults(cloudConfigMap models.CloudConfig, publicIPMap map[string]string, ansibleHostIDs []string) {
+// printResults prints the results of setting up Avalanche nodes.
+func printResults(
+	cloudConfigMap models.CloudConfig,
+	publicIPMap map[string]string,
+	ansibleHostIDs []string,
+) {
+	ux.Logger.PrintToUser("")
 	ux.Logger.PrintToUser("======================================")
 	ux.Logger.PrintToUser("AVALANCHE NODE(S) SUCCESSFULLY SET UP!")
 	ux.Logger.PrintToUser("======================================")
 	ux.Logger.PrintToUser("Please wait until the node(s) are successfully bootstrapped to run further commands on the node(s)")
 	ux.Logger.PrintToUser("")
 	ux.Logger.PrintToUser("Here are the details of the set up node(s): ")
 	for region, cloudConfig := range cloudConfigMap {
-		ux.Logger.PrintToUser(fmt.Sprintf("Don't delete or replace your ssh private key file at %s as you won't be able to access your cloud server without it", cloudConfig.CertFilePath))
+		ux.Logger.PrintToUser("Region: [%s] ", region)
+		if len(cloudConfig.APIInstanceIDs) > 0 {
+			ux.Logger.PrintToUser("")
+			ux.Logger.PrintToUser("======================================")
+			ux.Logger.PrintToUser("API Endpoint(s) for region [%s]: ", region)
+			for _, apiNode := range cloudConfig.APIInstanceIDs {
+				ux.Logger.PrintToUser("    http://%s:9650", publicIPMap[apiNode])
+			}
+			ux.Logger.PrintToUser("======================================")
+			ux.Logger.PrintToUser("")
+		}
+		ux.Logger.PrintToUser("Don't delete or replace your ssh private key file at %s as you won't be able to access your cloud server without it", cloudConfig.CertFilePath)
 		for i, instanceID := range cloudConfig.InstanceIDs {
-			publicIP := ""
-			publicIP = publicIPMap[instanceID]
 			ux.Logger.PrintToUser("======================================")
-			ux.Logger.PrintToUser(fmt.Sprintf("Node %s details: ", ansibleHostIDs[i]))
-			ux.Logger.PrintToUser(fmt.Sprintf("Cloud Instance ID: %s", instanceID))
-			ux.Logger.PrintToUser(fmt.Sprintf("Public IP: %s", publicIP))
-			ux.Logger.PrintToUser(fmt.Sprintf("Cloud Region: %s", region))
+			if slices.Contains(cloudConfig.APIInstanceIDs, instanceID) {
+				ux.Logger.PrintToUser("API node %s details: ", ansibleHostIDs[i])
+			} else {
+				ux.Logger.PrintToUser("Stake node %s details: ", ansibleHostIDs[i])
```

Comment:
if on fuji/mainnet, it may not be yet a stake node until it is added as validator.
Maybe just print "node %s details" in general, and "(api) node %s details" if devnet api

Reply:
agree

Reply:
yes